datanommer.models/alembic/versions/1d4feffd78fe_add_historic_user_an.py

```diff
@@ -83,15 +83,15 @@ def upgrade():
 
         if not msg.users:
             new_usernames = msg2usernames(msg.__json__(), **config)
-            print "Updating users to %r" % new_usernames
+            print("Updating users to %r" % new_usernames)
```

Comment:
As long as only one argument is used, that's not necessary.

If compatibility with Python 2.5 is not needed, I can add the import. I didn't find much info on what Python versions need to be supported, so I was conservative.